src/components/FileManager/FileManager.tsx

```diff
@@ -1129,6 +1143,28 @@ export const DialFileManagerView: FC = () => {
     [renamedPath, handleTableRowClick],
   );
 
+  const emptyStateRenderer = useCallback(
+    () => (
+      <DialNoDataContent
+        title={emptyStateTitle}
+        description={emptyStateDescription}
+        descriptionClassName="text-sm"
+        containerClassName="gap-3 absolute size-full bg-layer-2 border rounded-[4px] border-primary z-40"
```

Comment:
Are we sure we need `absolute` positioning here? Maybe instead of rendering the `Grid` we should just show an empty state.